pelican/tools/templates/tasks.py.jinja2

```diff
@@ -99,23 +99,31 @@ def preview(c):
 def livereload(c):
     """Automatically reload browser tab upon file modification."""
     from livereload import Server
-    build(c)
+
+    def cached_build():
+        cmd = '-s {settings_base} -e CACHE_CONTENT=True LOAD_CONTENT_CACHE=True'
+        pelican_run(cmd.format(**CONFIG))
+
+    cached_build(c)
```

Comment:
Since `cached_build()` was defined as taking no arguments, won't invoking the function with `cached_build(c)` yield an error? Something like:
```
TypeError: cached_build() takes 0 positional arguments but 1 was given
```

Reply:
Right!